app/models/user.rb

```diff
@@ -46,6 +48,9 @@ class User < ApplicationRecord
   attribute :problem_settings, :json
   attribute :file_list_settings, :json
 
+  attribute :quota_use_site_default, :boolean, default: true
+  attribute :quota, :integer
```

Comment:
I don't think these attribute lines are required; the ones above are only because we needed to provide a JSON hint to MariaDB; these will be fine without.